packages/components/src/button/style.scss

```diff
@@ -196,6 +196,11 @@
 	&.is-tertiary {
 		color: theme(outlines);
 
+		// Matches default button in hit area.
```

Comment:
Might be handy to document how you got these values, I'm guessing they're based on the size of these tertiary buttons and the gap between them and the default button's size?

Reply:
It's from https://github.com/WordPress/gutenberg/blob/197467fbc243a46f8e759d24fc838c36e4e650f0/packages/components/src/button/style.scss#L11.

What would be a good comment to point to that?

Reply:
Huh, so they're basically the same values (and should stay linked, right?)

Using/adding a variable so they don't go out of sync would be good; otherwise you can just reference the file name/line number directly (that GitHub link is fine, even).

Is the lack of `1px` `padding-bottom` intentional?

Reply:
I'll look at adding a variable, or addressing the line number, or both.

Yes the lack of bottom padding is intentional, it's there for the other button styles because they have a thicker bottom border to imply bevel with topdown shadow, and the bottom padding does something to something there.

Reply:
I clarified the comment a bit, but decided to not variableize anything for now. Reason being, right now we're in that place where some styles are inherited from upstream WordPress styles, and it's a bit messy. We want to move to a place where things are more componentized, and that will require a refactor which should make this file much simpler.